scripts/hid_configurator/configurator.py

```diff
@@ -242,12 +291,13 @@ def open_device(device_type):
         print("Unsupported device type")
         return None
 
+    vid = DEVICE[device_type]['vid']
     try:
-        dev = hid.Device(vid=NORDIC_VID, pid=get_device_pid(device_type))
+        dev = hid.Device(vid=vid, pid=get_device_pid(device_type))
         print("Found device")
     except hid.HIDException:
         try:
-            dev = hid.Device(vid=NORDIC_VID, pid=get_device_pid('dongle'))
+            dev = hid.Device(vid=vid, pid=get_device_pid('dongle'))
```

Comment:
We need dongle vid here (it may differ from device vid)

Reply:
Fixed